### PR TITLE
Fix params mismatch and redundant `.softmax()` in FlashDeformAttn.

### PR DESCRIPTION
**Pull request by HELLORPG**
Fix two issues in FlashDeformAttn:

1. The mismatch of params for FlashDeformAttnFunction.
2. The redundant usage of `F.softmax()` in FlashDeformAttn.

Ack.
Discussed with @WANGSSSSSSS

### CODE REVIEW

**Comment by YuwenXiong** [comment]
Thanks for the fix!
